builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -76,10 +76,17 @@ local function start_install(calling_dialog, package)
 			if not path then
 				gamedata.errormessage = msg
 			else
+				print("Successfully installed to " .. path)
```

Comment:
Needs removal

Reply:
(with a context, we don't know what is installed :p)